mlflow/tracking/fluent.py

```diff
@@ -1086,7 +1087,10 @@ def log_params(
 
 
 def log_input(
-    dataset: Dataset, context: Optional[str] = None, tags: Optional[Dict[str, str]] = None
+    dataset: Dataset,
+    context: Optional[str] = None,
+    tags: Optional[Dict[str, str]] = None,
+    inputs: Optional[List[ModelInput]] = None,
```

Comment:
```suggestion
    model: Optional[ModelInput] = None,
```

This should be `model`.